### PR TITLE
Resolve #4739

### LINKED ISSUE #4739: update `--server-name` entry in `--help`
It looks like the `--help` text for `--server-name` was not updated when we switched to hostname-based auto-names.

### PR DESCRIPTION
**Pull request by pritambaral**
Update `--server-name` explanation text in the output of `--help`

### CODE REVIEW

File: src/clustering/administration/main/command_line.cc [hunk 1]
```diff
@@ -891,7 +891,7 @@ options::help_section_t get_server_options(std::vector<options::option_t> *optio
                                              options::OPTIONAL));
     help.add("-n [ --server-name ] arg",
              "the name for this server (as will appear in the metadata).  If not"
-             " specified, it will be randomly chosen from a short list of names.");
+             " specified, one will be generated from the hostname and a random alphanumeric string.");
```
Comment (danielmewes) [inline src/clustering/administration/main/command_line.cc]: Sounds good.
One minor thing: Could you break this string up such that the line is at most 89 characters long?

**Comment by danielmewes** [comment]
Thanks @pritambaral. I left one small comment and then we can merge it.

**Comment by pritambaral** [comment]
String has been broken up

File: src/clustering/administration/main/command_line.cc [hunk 2]
```diff
@@ -891,7 +891,8 @@ options::help_section_t get_server_options(std::vector<options::option_t> *optio
                                              options::OPTIONAL));
     help.add("-n [ --server-name ] arg",
              "the name for this server (as will appear in the metadata).  If not"
-             " specified, it will be randomly chosen from a short list of names.");
+			 " specified, one will be generated from the hostname and a random "
```
Comment (danielmewes) [inline src/clustering/administration/main/command_line.cc]: One last thing: Could you make the indentation match the one of the line above ("the name for this ...")? We usually don't add additional indentation for strings that go over multiple lines.

Comment (pritambaral) [inline src/clustering/administration/main/command_line.cc]: Ah, looks like my non-standard editor conifg is leaking. Sorry about that. :)

**Comment by pritambaral** [comment]
Indentation fixed

**Comment by danielmewes** [comment]
Looks good, thanks @pritambaral

**Comment by danielmewes** [comment]
Cherry-picked into v2.1.x with 82c745eadb4c514ffb0ab1039144f672df4439f9

**Pull request merged**